cmd/ndm_daemonset/controller/blockdevicestore.go

```diff
@@ -38,7 +39,9 @@ func (c *Controller) CreateBlockDevice(blockDevice apis.BlockDevice) error {
 	blockDevice.SetNamespace(c.Namespace)
 
 	blockDeviceCopy := blockDevice.DeepCopy()
-	err := c.Clientset.Create(context.TODO(), blockDeviceCopy)
+	err := c.Clientset.Create(context.TODO(), blockDeviceCopy, &client.CreateOptions{
+		FieldManager: "NDM",
```

Comment:
Is this required on every create call? Also lower case is preferred.
```suggestion
		FieldManager: "ndm",
```